flink-connector-elasticsearch7/pom.xml

```diff
@@ -81,20 +81,6 @@ under the License.
 			<groupId>org.elasticsearch.client</groupId>
 			<artifactId>elasticsearch-rest-high-level-client</artifactId>
 			<version>${elasticsearch.version}</version>
-			<!-- elasticsearch-rest-client includes two different versions of httpcore-nio -->
-			<exclusions>
-				<exclusion>
-					<groupId>org.apache.httpcomponents</groupId>
-					<artifactId>httpcore-nio</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
-
-		<!-- We need to include httpcore-nio again in the correct version due to the exclusion above -->
-		<dependency>
```

Comment:
Without this and the exclusion the original Flink CI would fail with a dependencies convergence error.
You can run in the original repo with `mvn validate -Pcheck-convergence` and maven 3.2.5.
The issue is that their client bundles two versions of the httpcomponents. I would say such dependency convergence test is in general a good idea.

Reply:
Thanks. I'll also add that to the pipeline, that must be there

Reply:
@afedulov FYI https://github.com/apache/flink-connector-elasticsearch/pull/16

Reply:
Resolved